home/templates/home/classroom_detail.html

```diff
@@ -1,32 +1,169 @@
 {%  extends "home/home.html" %}
+{% load crispy_forms_tags %}
 
 {% block content %}
+
 <style>
 h2 {
-    color: firebrick;
+    color: steelblue;
     padding-left: 10px;
 }
 strong {
+    font-size: 24px;
     padding-left: 10px;
+    padding-bottom: 1px;
 }
 p {
+    font-size: 16px;
     padding-left: 20px;
     padding-top: 2px;
 }
+.button {
+   font-size: 10px;
+   font-family: "American Typewriter";
+   border-radius: 14px;
+   background-color: white;
+   color: #4CAF50;
+   border: 2px solid #4CAF50;
+}
 </style>
 
-<h2>{{ object.course }}</h2>
-<strong> Teacher: </strong>
-<p>{{ object.teacher.first_name }}  {{ object.teacher.last_name }} </p>
-<strong> Teacher Assistant: </strong>
-<p>{{ object.teacher_assistant.first_name }} {{ object.teacher_assistant.last_name }}</p>
-<strong> Volunteers: </strong>
+<script>
+function ChangeTeacherForm() {
```

Comment:
Using a class or classes to control visibility would be a more common practice here, then you could have this work a bit more declaratively with CSS. I would suggest that we use a library for UI (e.g. React, Vue, or Svelte) rather than to do everything from scratch.

Reply:
Can I do that in August?  I agree that the front end needs a lot of work, and this is a particularly ugly case of JS in HTML, but I've never used anything but HTML/Bootstrap so it might take me a week to get used to.

Reply:
Absolutely